modules/minimum-required-permissions-ipi-gcp-xpn.adoc

```diff
@@ -7,9 +7,12 @@
 
 When you are installing a cluster to a link:https://cloud.google.com/vpc/docs/shared-vpc[shared VPC], you must configure the service account for both the host project and the service project. If you are not installing to a shared VPC, you can skip this section.
 
-You must apply the minimum roles required for a standard installation as listed above, to the service project. Note that custom roles, and therefore fine-grained permissions, cannot be used in shared VPC installations because GCP does not support adding the required permission `compute.organizations.administerXpn` to custom roles.
```

Comment:
QUERY TO SME: I do not see any minimum permissions listed "above". Am I missing something in the [published docs](https://docs.openshift.com/container-platform/4.13/installing/installing_gcp/installing-gcp-account.html#minimum-required-permissions-ipi-gcp-xpn)

I do see a table below the paragraph named "Required minimal permissions" if this is what was meant?

Reply:
@dfitzmau Let me clarify a bit. The section `Required permissions for creating firewalls in the host project` and the section `Required minimal permissions` are both talking about the required permissions in the host project, where the latter lists the minimal permissions in the host project, and the former lists the required permissions if user wants the installer creating firewall-rules in the host project during the installation.